### PR TITLE
create table with comment

### PR DESCRIPTION
**Pull request by qingxinhome**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

issue #3601

**What this PR does / why we need it:**

In 0.6, column comments and table comments of tables are supported when creating tables

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/frontend/mysql_cmd_executor.go [hunk 1]
```diff
@@ -234,7 +246,7 @@ func handleShowCreateTable(ses *Session) error {
 		if typ.Oid == types.T_varchar || typ.Oid == types.T_char {
 			typeStr += fmt.Sprintf("(%d)", d[charWidthPos].(int32))
 		}
-		createStr += fmt.Sprintf("`%s` %s %s", colName, typeStr, nullOrNot)
+		createStr += fmt.Sprintf("`%s` %s %s%s", colName, typeStr, nullOrNot, hasAttrComment)
```
Comment (jianwan0214) [inline pkg/frontend/mysql_cmd_executor.go:249]: you should add space between %s

**Review by daviszhen — APPROVED** [review]

**Review by ouyuanning — APPROVED** [review]

**Review by LeftHandCold — APPROVED** [review]
lgtm

File: pkg/sql/plan/build_ddl.go [hunk 2]
```diff
@@ -66,6 +66,11 @@ func buildCreateTable(stmt *tree.CreateTable, ctx CompilerContext) (*Plan, error
 			})
 		// todo confirm: option data store like this?
 		case *tree.TableOptionComment:
+			if getNumOfCharacters(opt.Comment) > maxLengthOfTableComment {
```
Comment (nnsgmsone) [inline pkg/sql/plan/build_ddl.go:69]: why not use engine.CommentDef?

Comment (qingxinhome) [reply pkg/sql/plan/build_ddl.go:69]: engine.CommentDef is used for table comment, not a comment for a column

File: pkg/sql/plan/build_ddl.go [hunk 3]
```diff
@@ -66,6 +66,11 @@ func buildCreateTable(stmt *tree.CreateTable, ctx CompilerContext) (*Plan, error
...
+				errmsg := fmt.Sprintf("Comment for table '%s' is too long (max = %d)", createTable.TableDef.Name, maxLengthOfTableComment)
+				return nil, errors.New(errno.InvalidOptionValue, errmsg)
+			}
+
 			properties := []*plan.Property{
```
[the remark this answers is not in the document]
Comment (qingxinhome) [reply pkg/sql/plan/build_ddl.go:74]: Indeed, kV is currently used to define the attributes of the storage table in the plan, which needs to be reconstructed

**Pull request merged**